infra/src/.terraform.lock.hcl

```diff
@@ -2,23 +2,22 @@
 # Manual edits may be lost in future updates.
 
 provider "registry.terraform.io/azure/azapi" {
-  version     = "1.11.0"
-  constraints = "<= 1.11.0"
+  version     = "1.15.0"
```

Comment:
Probably some provider is missing. Please run
`terraform providers lock -platform=windows_amd64 -platform=darwin_amd64 -platform=darwin_arm64 -platform=linux_amd64`
everytime the lock file changes

Reply:
thank's! done in f263d02475dda77824f935b5a941412ea1548686